### DISCUSSION TITLE
Progress log

### DISCUSSION THREAD

**Discussion by tolgaburcak** [inlife/nexrender#980] [General]
Hi,

I couldn't log the progress. I can see files in my tmp folder. It starts rendering, result.25320.22148.m4v file size is increasing slowly but I can't see the progress on console.log(). Do you have any suggestion about this?

code:
result.on('created', job => console.log('project has been created'))
result.on('started', job => console.log('project rendering started'))
result.on('progress', (job, percents) => console.log('project is at: ' + percents + '%'))
result.on('finished', job => console.log('project rendering finished'))
result.on('error', err => console.log('project rendering error', err))

cmd output:
project has been created
project rendering started
project is at: 100%
project rendering finished

worker output:
...
[-pH_niPRfVaEdp5lQclrS] rendering took ~83.706 sec.
...

Thanks

**Comment by inlife** [comment]
Most likely, it's related to the AE language used. Nexrender parses AE log files to measure time, and so it happens AE renders time stuff differently in some languages. Alternatively, your rendering composition uses frames instead of seconds to measure progress, which also influences how logs are written.

**Reply by tolgaburcak** [reply]
I changed the composition's timecode display style to seconds and the progress came back!

Thanks a lot!
